drake/CMakeLists.txt

```diff
@@ -125,7 +134,9 @@ file(WRITE ${PROJECT_SOURCE_DIR}/core/Path.h
     "    return root;\n"
     "  }\n"
     "} // end namespace Drake\n")
-
+set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${drake_path_header})
+install(FILES ${drake_path_header} DESTINATION include/drake/)
+include_directories(${generated_dir})
```

Comment:
same trick is happening here, but with `pod-build/generated/drake/Path.h` which gets installed to `build/include/drake/Path.h` for easy referencing later...